pages/guide.md

```diff
@@ -13,17 +13,18 @@ This page provides implementation guidance for [Binding Operational Directive 18
 
 ### Checklist
 
-* Within **30 days** of BOD issuance, submit an "Agency Plan of Action" to <FNR.BOD@hq.dhs.gov> and begin implementing the plan.
-* At **60 days** after BOD issuance (**and at every 30 days until full implementation**), provide a status report of plan implementation to <FNR.BOD@hq.dhs.gov>.
-* Within **90 days** of BOD issuance, configure all internet-facing mail servers to offer STARTTLS.
-* By **120 days** after BOD issuance:
+* Within **30 days** of BOD issuance *(November 15th, 2017)*, submit an "Agency Plan of Action" to <FNR.BOD@hq.dhs.gov> and begin implementing the plan.
+* At **60 days** *(December 15th, 2017)* after BOD issuance (**and at every 30 days until full implementation**), provide a status report of plan implementation to <FNR.BOD@hq.dhs.gov>.
+* Within **90 days** *(January 15, 2018)* of BOD issuance:
```

Comment:
Oct 16 + 90 days is actually January 14, not 15 (though I imagine this was shifted due to the 14th being a Sunday).

Reply:
Yeah, that date was shifted.

Reply:
Nod nod. Does that need to be called out explicitly do you think, or that is near enough and expected enough that would happen?

Reply:
Sorry, missed your comment before merging. 

The Nov. 15th (16th) date was already communicated out to agency security leads directly, and it's also _this week_. Also, not many know yet that these dates are on the sites; the chance of confusion seems small.🤞